com.iche2.gai.yaml

```
@@ -0,0 +1,45 @@
+app-id: com.iche2.gai
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: gai
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-music
+  - --filesystem=xdg-videos
+
+modules:
+  - name: gai
+    buildsystem: simple
+    build-commands:
+      - cp -R gai /app/gai
+      - cp -R lib /app/
+      - cp -R data /app/
+      - mkdir -p /app/bin/
+      - mkdir -p /app/share/icons/hicolor/scalable/apps/
+      - mkdir -p /app/share/metainfo/
+      - mkdir -p /app/share/applications/      
+      - ln -s /app/gai /app/bin/gai      
+      - cp /app/data/flutter_assets/assets/gai_icon_square_512x512.png /app/share/icons/hicolor/scalable/apps/com.iche2.gai.png
+      - cp /app/data/flutter_assets/assets/gai_icon_square_512x512.svg /app/share/icons/hicolor/scalable/apps/com.iche2.gai.svg
+      - cp /app/data/flutter_assets/assets/gai_appstream_metainfo.xml /app/share/metainfo/com.iche2.gai.metainfo.xml
+      - cp /app/data/flutter_assets/assets/gai_flatpak.desktop /app/share/applications/com.iche2.gai.desktop
```

PNG is not scalable.

```suggestion
      - ln -s /app/gai /app/bin/gai      
      - install -Dm644 /app/data/flutter_assets/assets/gai_icon_square_512x512.png /app/share/icons/hicolor/512x512/apps/com.iche2.gai.png
      - install -Dm644 /app/data/flutter_assets/assets/gai_icon_square_512x512.svg /app/share/icons/hicolor/scalable/apps/com.iche2.gai.svg
      - install -Dm644 /app/data/flutter_assets/assets/gai_appstream_metainfo.xml /app/share/metainfo/com.iche2.gai.metainfo.xml
      - install -Dm644 /app/data/flutter_assets/assets/gai_flatpak.desktop /app/share/applications/com.iche2.gai.desktop
```

---

SVG desktop icon updated.

---

the suggestion was ignored...

---

can you simplify this? mkdir and cp is not needed, install can do both.

---

I'm new and unsure if the `install` command can recursively copy all files like `cp -R`.  @bbhtt. Thanks for the guidance,  I'll use `install` in my next update.

---

No, this is about the last 3 cp commands that you are using to install icon desktop and metainfo.

---

YAML updated; file copy changed to `install`.